samples/javascript_nodejs/42.scale-out/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "scale-out",
+    "version": "1.0.0",
+    "description": "Demonstrates how you can build your own state solution from the ground up that supports scaled out deployment with ETag based locking.",
+    "author": "Generated using Microsoft Bot Builder Yeoman generator v4.22.1",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "start": "node ./index.js",
+        "watch": "nodemon ./index.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "@microsoft/recognizers-text-data-types-timex-expression": "1.1.4",
```

Comment:
Is this used?  I could have missed it.

Reply:
Removed it as it is not used in this sample